{"uid":"8ae0dd726f96a3c3","name":"Login with invalid credentials test","fullName":"tests.login_test.TestLogin#test_invalid_login","historyId":"d8696551571014a71a874c85cadaf3fe","time":{"start":1705626886820,"stop":1705626897089,"duration":10269},"description":"invalid login","descriptionHtml":"<p>invalid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5606c72f4f83 <unknown>\n#1 0x5606c6fadcf7 <unknown>\n#2 0x5606c6ffd99e <unknown>\n#3 0x5606c6ffdaa1 <unknown>\n#4 0x5606c7048d64 <unknown>\n#5 0x5606c70270dd <unknown>\n#6 0x5606c7046006 <unknown>\n#7 0x5606c7026e53 <unknown>\n#8 0x5606c6feedd4 <unknown>\n#9 0x5606c6ff01de <unknown>\n#10 0x5606c72b9531 <unknown>\n#11 0x5606c72bd455 <unknown>\n#12 0x5606c72a5f55 <unknown>\n#13 0x5606c72be0ef <unknown>\n#14 0x5606c728999f <unknown>\n#15 0x5606c72e2008 <unknown>\n#16 0x5606c72e21d7 <unknown>\n#17 0x5606c72f4124 <unknown>\n#18 0x7f0c3ee94ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f2de1eef560>\nemail = 'nirt236@gmail.com', password = '123456'\ndata = Data(forgot_password=ForgotPassword(error_message=\"We can't find a user with that e-mail address.\", success_message='A...iteria', non_existing_project='Non Existing', project_name='test project', project_type='quiz', template_type='Blank'))\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, data: Data):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"ad2c947b61ff46d9fd174608d6b24399\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f2de1e596c0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5606c72f4f83 <unknown>\nE       #1 0x5606c6fadcf7 <unknown>\nE       #2 0x5606c6ffd99e <unknown>\nE       #3 0x5606c6ffdaa1 <unknown>\nE       #4 0x5606c7048d64 <unknown>\nE       #5 0x5606c70270dd <unknown>\nE       #6 0x5606c7046006 <unknown>\nE       #7 0x5606c7026e53 <unknown>\nE       #8 0x5606c6feedd4 <unknown>\nE       #9 0x5606c6ff01de <unknown>\nE       #10 0x5606c72b9531 <unknown>\nE       #11 0x5606c72bd455 <unknown>\nE       #12 0x5606c72a5f55 <unknown>\nE       #13 0x5606c72be0ef <unknown>\nE       #14 0x5606c728999f <unknown>\nE       #15 0x5606c72e2008 <unknown>\nE       #16 0x5606c72e21d7 <unknown>\nE       #17 0x5606c72f4124 <unknown>\nE       #18 0x7f0c3ee94ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"base_url","time":{"start":1705626854491,"stop":1705626854491,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"data","time":{"start":1705626854491,"stop":1705626854492,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1705626854491,"stop":1705626854491,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"invalid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5606c72f4f83 <unknown>\n#1 0x5606c6fadcf7 <unknown>\n#2 0x5606c6ffd99e <unknown>\n#3 0x5606c6ffdaa1 <unknown>\n#4 0x5606c7048d64 <unknown>\n#5 0x5606c70270dd <unknown>\n#6 0x5606c7046006 <unknown>\n#7 0x5606c7026e53 <unknown>\n#8 0x5606c6feedd4 <unknown>\n#9 0x5606c6ff01de <unknown>\n#10 0x5606c72b9531 <unknown>\n#11 0x5606c72bd455 <unknown>\n#12 0x5606c72a5f55 <unknown>\n#13 0x5606c72be0ef <unknown>\n#14 0x5606c728999f <unknown>\n#15 0x5606c72e2008 <unknown>\n#16 0x5606c72e21d7 <unknown>\n#17 0x5606c72f4124 <unknown>\n#18 0x7f0c3ee94ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f2de1eef560>\nemail = 'nirt236@gmail.com', password = '123456'\ndata = Data(forgot_password=ForgotPassword(error_message=\"We can't find a user with that e-mail address.\", success_message='A...iteria', non_existing_project='Non Existing', project_name='test project', project_type='quiz', template_type='Blank'))\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, data: Data):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"ad2c947b61ff46d9fd174608d6b24399\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f2de1e596c0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5606c72f4f83 <unknown>\nE       #1 0x5606c6fadcf7 <unknown>\nE       #2 0x5606c6ffd99e <unknown>\nE       #3 0x5606c6ffdaa1 <unknown>\nE       #4 0x5606c7048d64 <unknown>\nE       #5 0x5606c70270dd <unknown>\nE       #6 0x5606c7046006 <unknown>\nE       #7 0x5606c7026e53 <unknown>\nE       #8 0x5606c6feedd4 <unknown>\nE       #9 0x5606c6ff01de <unknown>\nE       #10 0x5606c72b9531 <unknown>\nE       #11 0x5606c72bd455 <unknown>\nE       #12 0x5606c72a5f55 <unknown>\nE       #13 0x5606c72be0ef <unknown>\nE       #14 0x5606c728999f <unknown>\nE       #15 0x5606c72e2008 <unknown>\nE       #16 0x5606c72e21d7 <unknown>\nE       #17 0x5606c72f4124 <unknown>\nE       #18 0x7f0c3ee94ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1705626886820,"stop":1705626897088,"duration":10268},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5606c72f4f83 <unknown>\n#1 0x5606c6fadcf7 <unknown>\n#2 0x5606c6ffd99e <unknown>\n#3 0x5606c6ffdaa1 <unknown>\n#4 0x5606c7048d64 <unknown>\n#5 0x5606c70270dd <unknown>\n#6 0x5606c7046006 <unknown>\n#7 0x5606c7026e53 <unknown>\n#8 0x5606c6feedd4 <unknown>\n#9 0x5606c6ff01de <unknown>\n#10 0x5606c72b9531 <unknown>\n#11 0x5606c72bd455 <unknown>\n#12 0x5606c72a5f55 <unknown>\n#13 0x5606c72be0ef <unknown>\n#14 0x5606c728999f <unknown>\n#15 0x5606c72e2008 <unknown>\n#16 0x5606c72e21d7 <unknown>\n#17 0x5606c72f4124 <unknown>\n#18 0x7f0c3ee94ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 60, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"5b9788b6af968c10","name":"public ip address","source":"5b9788b6af968c10.txt","type":"text/plain","size":12},{"uid":"38da5851c059f772","name":"Cookies","source":"38da5851c059f772.json","type":"application/json","size":3872},{"uid":"2d2503d87ac5eba8","name":"Session Storage","source":"2d2503d87ac5eba8.json","type":"application/json","size":359},{"uid":"d5ab225bafda918f","name":"Local Storage","source":"d5ab225bafda918f.json","type":"application/json","size":208},{"uid":"3e787679d1017080","name":"Console Logs","source":"3e787679d1017080.json","type":"application/json","size":2},{"uid":"4c120656501c5af5","name":"Full Page Screenshot","source":"4c120656501c5af5.png","type":"image/png","size":5704997},{"uid":"68d33602c8a9aae8","name":"URL","source":"68d33602c8a9aae8.uri","type":"text/uri-list","size":23},{"uid":"c88bd9e3f27bc963","name":"Network Logs","source":"c88bd9e3f27bc963.json","type":"application/json","size":23204}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"feature","value":"Login"},{"name":"severity","value":"blocker"},{"name":"epic","value":"Security"},{"name":"tag","value":"run(order=3)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"login_test"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az1538-683"},{"name":"thread","value":"2341-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.login_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"email","value":"'nirt236@gmail.com'"},{"name":"password","value":"'123456'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["run(order=3)","security"]},"source":"8ae0dd726f96a3c3.json","parameterValues":["'nirt236@gmail.com'","'123456'"]}